particle-builtins/src/builtins.rs

```diff
@@ -706,14 +706,15 @@ where
             .modules
             .add_module_from_vault(module_path, config, params)?;
 
-        Ok(JValue::String(module_hash))
+        Ok(json!(module_hash))
     }
 
     fn add_blueprint(&self, args: Args) -> Result<JValue, JError> {
         let mut args = args.function_args.into_iter();
-        let blueprint_request: AddBlueprint = Args::next("blueprint_request", &mut args)?;
-
-        let blueprint_id = self.modules.add_blueprint(blueprint_request)?;
+        let blueprint_request: String = Args::next("blueprint_request", &mut args)?;
+        let blueprint = AddBlueprint::decode(blueprint_request.as_bytes())
```

Comment:
https://github.com/fluencelabs/aqua-lib/blob/main/builtin.aqua#LL309C1-L310C53

aqua-lib calls it `blueprint`, so we gotta match names
```
add_blueprint(blueprint: AddBlueprint) -> string
```

```suggestion
        let blueprint: String = Args::next("blueprint", &mut args)?;
        let blueprint = AddBlueprint::decode(blueprint.as_bytes())
```